packages/core-components/src/lib/atoms/inputs/date-range/DateRange.svelte

```diff
@@ -0,0 +1,95 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script lang="ts">
+	import type { Writable } from 'svelte/store';
+
+	import DateRange from './_DateRange.svelte';
+	import { getContext } from 'svelte';
+	import { INPUTS_CONTEXT_KEY } from '@evidence-dev/component-utilities/globalContexts';
+	import type { QueryStore } from '@evidence-dev/query-store';
+	import { buildQuery } from '@evidence-dev/component-utilities/buildQuery';
+	import { getLocalTimeZone } from '@internationalized/date';
+	import HiddenInPrint from '../shared/HiddenInPrint.svelte';
+	import { page } from '$app/stores';
+
+	function dateToYYYYMMDD(date: Date) {
+		return date.toISOString().split('T')[0];
+	}
+
+	const inputs: Writable<object> = getContext(INPUTS_CONTEXT_KEY);
+
+	export let name: string;
+	export let title: string | undefined;
+	export let hideDuringPrint = true;
```

Comment:
false?

Reply:
I thought consensus was default to hiding during print `Hide inputs on print, incl. Print option default to false` but I might be misinterpreting

Reply:
yep you are right